tutorials/Python/00_Introduction/Introduction.ipynb

```diff
@@ -35,22 +36,23 @@
     "8. Documentation\n",
     "9. Libraries\n",
     "10. Numpy\n",
-    "11. Analysing MD\n",
+    "11. OOP (Object Orientated Programming)\n",
+    "12. Analysing MD\n",
     "\n",
-    "These notebooks are contained within their own labelled/named directories. Within these directories you will also find `datafiles` sub-directories that contain all the relevant data to run the notebook, in addition to copies of the notebooks with the exercise solutions (whose name ends in `_solutions.ipynb`).\n",
+    "These notebooks are contained within their own labelled/named directories. Within these directories you will also sometimes find `datafiles` sub-directories that contain all the relevant data to run the notebook, in addition to copies of the notebooks with the exercise solutions (whose name ends in `_solutions.ipynb`).\n",
     "\n",
     "We recommend that each section be followed in order, however those with prior experience with python may wish to jump to the later sections.\n",
     "\n",
     "### Launching the notebooks\n",
     "\n",
-    "When launching the notebooks locally/from a terminal (you will be doing this if you are attending the tutorial in person), do the following:\n",
-    "1. Open a terminal window (find the program `Terminal` and open it).\n",
-    "2. Activate the relevant python environment using: `conda activate oxpy`.\n",
-    "3. Navigate to the OxCompBio python tutorial in the terminal.\n",
-    "4. Type `jupyter notebook`. This will open an interactive jupyter session in your browser. You will then be able to nagivate through the different directories.\n",
-    "5. To launch a notebook, just click on it to start the session.\n",
-    "6. You can close the jupyter notebook via the dropdown menu within the notebook: `File > Close and Halt`. Alternatively you can close the entire jupyter session by typing `Ctl+C` in your terminal window.\n",
-    "7. You can then close the terminal, or re-start a jupyter session by following the above instructions from step 3.\n",
+    "Please see the `setup.md` \"Starting the tutorial\" instructions.\n",
```

Comment:
Should we mention the following commands need to run on a terminal? What about Windows?

Reply:
Done (see below)